runelite-client/src/main/java/net/runelite/client/plugins/objectindicators/ObjectIndicatorsPlugin.java

```diff
@@ -116,10 +116,17 @@ ObjectIndicatorsConfig provideConfig(ConfigManager configManager)
 		return configManager.getConfig(ObjectIndicatorsConfig.class);
 	}
 
+	private static boolean reloadMapPointsOnStartup = false;
+
 	@Override
 	protected void startUp()
 	{
 		overlayManager.add(overlay);
+		if (reloadMapPointsOnStartup)
+		{
+			reloadMapPoints();
```

Comment:
You can't be calling client methods from EDT. This must be wrapped in `clientThread.invokeLater()`.